src/simulator_core/solver/network/assets/base_asset.py

```diff
@@ -93,6 +93,16 @@ def connect_node(self, connection_point: int, node: BaseNodeItem) -> None:
 
         self.connected_nodes[connection_point] = node
 
+    def disconnect_node(self, connection_point: int) -> None:
+        """Disconnects a node from a connection point of the asset.
+
+        :param connection_point: The index of the connection point to disconnect.
+        :type connection_point: int
+        :raises ValueError: If the connection point is not connected to a node.
+        """
+        self.check_connection_point_valid(connection_point)
```

Comment:
Zou bij de functie check_connection_point_valid de return ook niet None moeten zijn? Hoe wordt er nu met de fout omgegaan?

Reply:
Response from Copilot: 

The return type of the check_connection_point_valid function in your base_asset.py file is bool. This is because the function either returns True or raises an IndexError.

In Python, it's common to indicate that a function raises an exception in its docstring, but this doesn't affect the function's return type. The return type is determined by the type of the value that the function returns, not by the exceptions it might raise.

So, in your function's signature, you can specify the return type as bool:

This indicates that the function returns a boolean value. If the function raises an exception, it doesn't return a value.